sdks/java/harness/src/main/java/org/apache/beam/runners/core/BeamFnDataWriteRunner.java

```diff
@@ -37,6 +41,10 @@
  * For each request, call {@link #registerForOutput()} to start and call {@link #close()} to finish.
  */
 public class BeamFnDataWriteRunner<InputT> {
+  private final ObjectMapper mapper =
```

Comment:
You can use a plain ObjectMapper since we only need to support custom user types on PipelineOptions